src/main/proto/ga4gh/genotype_phenotype_service.proto

```diff
@@ -80,28 +89,34 @@ message EvidenceQuery {
     // ECO or OBI is recommended
     OntologyTerm evidenceType = 1;
 
-    string description = 2;  /*regex*/
+    // regex
+    string description = 2;
 
-    repeated ExternalIdentifier external_identifiers = 3; /* new field */
+    // new field
+    repeated ExternalIdentifier external_identifiers = 3;
 }
 
 message SearchPhenotypesRequest {
     // The `PhenotypeAssociationSet` to search. Mandatory
     string phenotype_association_set_id = 1;
 
+    // Phenotype ID
     string id = 2;
 
-    string description = 3; /* The system should support regular expression matches */
+    // The system should support regular expression matches
```

Comment:
Two other options: remove the regex from the specification entirely as I believe we can satisfy the interchange use cases in its absence, or, allow regex matching on strings through the API. Just stating "regex" matching is probably not clear enough either, perhaps point to a more specific implementation of regular expressions? What features can they trust will be present? Grouping? Quantification?